app.rb

```diff
@@ -33,7 +33,7 @@ def list_all_people
   end
 
   def create_a_person
-    print 'Do you want to create a student (1) o a Teahcer (2)? [Input the number]: '
+    print 'Do you want to create a Student (1) or a Teacher (2)? [Input the number]: '
     option = gets.chomp.to_i
 
     case option
```

Comment:
- You have done great so far but there are few changes you need to make before moving to the next milestone. :smile: :+1: 
- [ ] Follwoing the [SOLID](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid`) PRINCIPLE AND DRY. You need to refactore your `create_student and create_teacher` method. All requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.
-  Also the input for `Age, name` is repeated in create person and teacher which doesn't follow the dry principle. I'll suggest you take note of these and update your code. :smile: :wave: